front/temporal/scrub_workspace/workflows.ts

```diff
@@ -19,36 +19,10 @@ const { scrubWorkspaceData, pauseAllConnectors } = proxyActivities<
   startToCloseTimeout: "60 minutes",
 });
 
-// DEPRECATED
-// TODO(@fontanierh): remove this workflow once the new one is deployed and no instances are still running.
 export async function scheduleWorkspaceScrubWorkflow({
   workspaceId,
 }: {
   workspaceId: string;
-}): Promise<boolean> {
```

Comment:
Can you provide some context here?

Reply:
It's a long-running workflow with timers and I had to make changes to it. So I had created a "v2", and planned to delete the v1 when all the v1s were done running. Now is the time to delete v1 since it's no longer running.

Reply:
thanks.